src/imagebuilder/azext_imagebuilder/_params.py

```diff
@@ -0,0 +1,91 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+# pylint: disable=too-many-lines
+# pylint: disable=too-many-statements
+
+from azure.cli.core.commands.parameters import (
+    tags_type,
+    get_enum_type,
+    resource_group_name_type,
+    get_location_type
+)
+from azure.cli.core.commands.validators import get_default_location_from_resource_group
+from knack.arguments import CLIArgumentType
+
+
+def load_arguments(self, _):
+
+    name_arg_type = CLIArgumentType(options_list=['--name', '-n'], metavar='NAME')
+    image_template_name = CLIArgumentType(overrides=name_arg_type,
+                                          help='The name of the image Template', id_part='name')
+
+    with self.argument_context('imagebuilder create') as c:
+        c.argument('resource_group', resource_group_name_type)
+        c.argument('image_template_name', image_template_name)
+        c.argument('location', arg_type=get_location_type(self.cli_ctx),
+                   validator=get_default_location_from_resource_group)
+        c.argument('tags', tags_type)
+        c.argument('customize', id_part=None, help='Specify the properties used to describe the customization steps of the image, like Image source etc')
```

Comment:
call out it could be uri of local file/remote uri, or text  directly?

Reply:
text directly, or @{filename}.
It's a good idea to support URI.